test/e2e/network/kube_proxy.go

```diff
@@ -105,7 +109,7 @@ var _ = SIGDescribe("Network", func() {
 						Image:           kubeProxyE2eImage,
 						ImagePullPolicy: "Always",
 						Args: []string{
-							"net", "--serve", fmt.Sprintf("0.0.0.0:%d", testDaemonHTTPPort),
```

Comment:
this is a curious change... aren't these equivalent?

Reply:
indeed, we've discussed this in other PR previously, but I keep thinking that 0.0.0.0 is misleading and this way is easy to understand that it listens in all addresses and families.
Just a matter of taste I guess

Reply:
They literally mean the same thing, right?

Reply:
Jordan clarified it to me 2 weeks ago https://github.com/kubernetes/kubernetes/issues/86479#issuecomment-567968306 , I've been thinking until then that unspecified addresses were IP family bounded 😅